### ISSUE TITLE
bio:position is a foaf:Person?

### ISSUE THREAD

**Issue by MidnightLightning** [iand/vocab-bio#5]
The description of `bio:position` [indicates](https://github.com/iand/vocab-bio/blob/master/schema.rdf#L757-L758) it's value is a `foaf:Person`, but does that make sense for that property? An "employment position or public office" sounds like it needs a different sort of Class altogether, one that indicates the name of the position, date it was first opened for applicants, other applicants to the position, etc.

**Comment by JBPressac** [comment]
I agree there is something weird here. But why using a Class ? For me bio:position values could be a litteral like "director", "employee", "engineer"...

**Comment by MidnightLightning** [comment]
True; I was thinking it could be a class since a `Position` could also have attributes like `salary` or `reportsTo`, and have things like `foaf:Project` objects related to it if you think about it as a child of a `foaf:Organization`.

However, in the context of `bio`, it's really just clarification for a `PositionChange` event or `Employment` event (domain should be limited to `PositionChange` or `Employment` classes, and not allow generic `Event`s?). Events start/end/bridge `Interval` classes, so maybe the `position` attribute should be a literal string on the `Interval` (or a new subclass of `Interval` for `EmployedAt`?) to indicate which position, and not be on the `Event` at all?

**Comment by JBPressac** [comment]
Hello,
Do you have examples ?

**Comment by MidnightLightning** [comment]
Examples of what? General `Event` classes bridging `Interval` classes are part of the diagrams as part of the `bio` project overall.
